src/store/src/InMemoryStore.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Store;
+
+use Symfony\AI\Platform\Vector\Vector;
+use Symfony\AI\Store\Document\VectorDocument;
+
+final class InMemoryStore implements VectorStoreInterface
+{
+    /**
+     * @var VectorDocument[]
+     */
+    private array $documents;
+
+    public function add(VectorDocument ...$documents): void
+    {
+        foreach ($documents as $document) {
+            $this->documents[] = $document;
+        }
```

Comment:
I think we could use `array_push` instead of iterating over all documents to add:
```suggestion
        array_push($this->documents, $documents);
```
see https://www.php.net/array_push